watson/cli.py

```diff
@@ -339,6 +341,9 @@ def report(watson, from_, to, projects, tags):
     `--tag` options. They can be specified several times each to add multiple
     projects or tags to the report.
 
+    You can change to output format from the console output to JSON with
+    specifying the `--json` argument.
```

Comment:
Again, the difference is not console vs something else but formatted text vs JSON.